infra/cifuzz/cifuzz.py

```diff
@@ -201,6 +211,55 @@ def run_fuzzers(fuzz_seconds, workspace):
   return True, False
 
 
+def get_lastest_build_version(project_name):
+  """Gets the latest OSS-Fuzz build version for a projects fuzzers.
+
+  Args:
+    project_name: The name of the project who's build is being retrieve.
+
+  Returns:
+    A string with the latest build version or None.
+  """
+  http_get_string = os.path.join(GCS_BASE_URL, project_name,
+                                 project_name + '-address-latest.version')
+  try:
+    response = urllib.request.urlopen(http_get_string)
+  except urllib.error.HTTPError:
+    logging.error('Error getting the lastest build version for %s.',
+                  project_name)
+    return None
+  return response.read().decode('UTF-8')
+
+
+def download_ossfuzz_build(project_name, out_dir):
+  """Downloads the latest OSS-Fuzz build from GCS.
+
+  Args:
+    project_name: The name of the relevant OSS-Fuzz project.
+    out_dir: The location where the build should be stored.
+
+  Returns:
+    A path to where the OSS-Fuzz build is located, or None.
+  """
+  if not os.path.exists(out_dir):
+    logging.error('Out directory %s does not exist.', out_dir)
+    return None
+  latest_build_str = get_lastest_build_version(project_name)
+  if not latest_build_str:
+    return None
+  build_dir = os.path.join(out_dir, 'ossfuzz_latest', project_name)
+  os.makedirs(build_dir, exist_ok=True)
+  http_get_string = os.path.join(GCS_BASE_URL, project_name, latest_build_str)
```

Comment:
posixpath instead of os.path.
Actually let's create a helper function url_join and do posixpath there.
Much clearer what we are doing than posixpath or os.path.